hack/install-router.sh

```diff
@@ -1,34 +1,71 @@
 #!/bin/bash
 set -e
 
-echo "Creating router file and starting pod..."
-
-# ID to be used as the k8s id and also appended to the container name
+# ID to be used as the k8s id and also appended to the container name. Defaults to router1
 ROUTER_ID="${1}"
-# IP address to connect to the master, :8080 will be automatically appended
-MASTER_IP="${2}"
+# Full address to connect to the master.
+MASTER_URL="${2}"
 # openshift executable - optional, will try to find it on the path if not specified
 OPENSHIFT="${3}"
 
 OS_ROOT=$(dirname "${BASH_SOURCE}")/..
 
+if [[ "${ROUTER_ID}" == "" ]]; then
+	echo "No router id provided, cannot create router..."
+	exit
+fi
+
+if [[ "${MASTER_URL}" == "" ]]; then
+	echo "No master url provided, cannot create router..."
+	exit
+fi
+
 if [[ "${OPENSHIFT}" == "" ]]; then
     if [[ "$(which osc)" != "" ]]; then
         OPENSHIFT=$(which osc)
     fi
 fi
 
-# update the template file
-cp ${OS_ROOT}/images/router/haproxy/pod.json /tmp/router.json
-sed -i s/MASTER_IP/${MASTER_IP}/ /tmp/router.json
-sed -i s/ROUTER_ID/${ROUTER_ID}/g /tmp/router.json
+OPENSHIFT_INSECURE="${OPENSHIFT_INSECURE:-false}"
+OPENSHIFT_CA_DATA="${OPENSHIFT_CA_DATA:-}"
+OPENSHIFT_CERT_DATA="${OPENSHIFT_CERT_DATA:-}"
+OPENSHIFT_KEY_DATA="${OPENSHIFT_KEY_DATA:-}"
```

Comment:
I tried setting these locally and starting the router with this message from the container:

`[vagrant@openshiftdev origin]$ docker logs 6e54
F0123 14:32:45.451188       1 router.go:46] Unable to configure Kubernetes client: crypto/tls: failed to parse certificate PEM data`